src/db/models/jaf.ts

```diff
@@ -0,0 +1,55 @@
+import { randomUUID } from "crypto";
+import sequelize from "sequelize";
+import { BelongsTo, Column, ForeignKey, Model, Table } from "sequelize-typescript";
+import { companyModel } from "./company";
+import { memberModel } from "./member";
+import { seasonModel } from "./season";
+
+@Table({
+  tableName: "jaf",
+})
+export class jafModel extends Model {
+  @Column({
+    primaryKey: true,
+    allowNull: false,
+    type: sequelize.UUID,
+    defaultValue: sequelize.UUIDV4,
+  })
+  id: typeof randomUUID;
+
+  @ForeignKey(() => seasonModel)
+  @Column(sequelize.UUID)
+  seasonId: typeof randomUUID;
+  @BelongsTo(() => seasonModel, "seasonId")
+  season: seasonModel;
+
+  @ForeignKey(() => memberModel)
+  @Column(sequelize.UUID)
+  recruiterId: typeof randomUUID;
+  @BelongsTo(() => memberModel, "memberId")
+  member: memberModel;
+
+  @ForeignKey(() => companyModel)
+  @Column(sequelize.UUID)
+  companyId: typeof randomUUID;
+  @BelongsTo(() => companyModel, "companyId")
+  company: companyModel;
+
+  @Column
+  role: string;
+
+  @Column
+  metadata: string;
+
+  @Column
+  docs: string;
+
+  @Column
+  accepted: boolean;
+
+  @Column
+  publicAccess: boolean;
+
+  @Column
+  eligibility: string;
+}
```

Comment:
Refer to [this](https://www.npmjs.com/package/sequelize-typescript#model-association) and add HasMany decorators to the table which contains the primary key for the foreign key referenced in some other table.
For eg:- 'Event' model uses jaf id, and the relation between Jaf and Event is one-to-many so use the BelongsTo decorator in Event model, and use the HasMany decorator in the Jaf model.
Do this for all model relations